examples/src/main/scala/com/http4s/rho/swagger/demo/StaticContentService.scala

```diff
@@ -1,28 +1,27 @@
 package com.http4s.rho.swagger.demo
 
 import cats.effect.{Blocker, ContextShift, IO}
-import org.http4s.{HttpRoutes, Request, Response, StaticFile}
 import org.http4s.dsl.io._
-
-import scala.concurrent.ExecutionContext.global
+import org.http4s.rho.RhoRoutes
+import org.http4s.{HttpRoutes, Request, Response, StaticFile}
 
 object StaticContentService {
   private val swaggerUiDir = "/swagger-ui"
 
-  def fetchResource(path: String, req: Request[IO])(implicit cs: ContextShift[IO]): IO[Response[IO]] = {
-    StaticFile.fromResource(path, Blocker.liftExecutionContext(global), Some(req)).getOrElseF(NotFound())
+  def fetchResource(path: String, req: Request[IO])(implicit cs: ContextShift[IO], blocker: Blocker): IO[Response[IO]] = {
```

Comment:
Again on this one, I know its an example, but lets set a good one, according to the [cats doc](https://typelevel.org/cats-effect/api/cats/effect/Blocker.html), instances of the `Blocker type`  ` should *not* be passed implicitly. `

Reply:
Personally I'm not convinced that the drawbacks of passing it as implicit overweight the convenience of doing it. That being said, I agree that the examples should rather stay in line with the official recommendation than my personal preferences ;).